collectors/python.d.plugin/nvidia_smi/nvidia_smi.chart.py

```diff
@@ -404,7 +404,7 @@ def data(self):
         for p in processes:
             data['process_mem_{0}'.format(p['pid'])] = p['used_memory']
             if p['username']:
```

Comment:
```python
if p['username'] and p['used_memory'] > 0:
```

Reply:
It is slightly different: in your way we not see the process (and the user) at all.
With my mod, I patch only user_nr ... 

I don't know which makes more sense...

Reply:
We would see the process.

This code that adds users
https://github.com/netdata/netdata/blob/0cb1d2a92e3aed369f32667128f65f68484267ca/collectors/python.d.plugin/nvidia_smi/nvidia_smi.chart.py#L406-L412

You said 

> I cannot consider a GPU used (by a user) with no memory allocated.

So we need to skip those users, right?

Reply:
we have to skip users for which used_memory is 0

`if p['used_memory'] > 0 : users.add(p['username'])`

Reply:
it is not becase `key = 'user_mem_{0}'.format(p['username'])`, what am i missing?

Reply:
process memory usage is `p['used_memory']`